src/internal/_flatCat.js

```diff
@@ -29,4 +29,4 @@ module.exports = (function() {
       }
     };
   };
-}());
+})();
```

Comment:
Never mind.  Looked it up and see this non-default preference in the config file above.  Will add a comment there.